common-crafting.lic

```diff
@@ -142,4 +142,13 @@ module DRCC
     end
     true
   end
+  
+  def logbook_item(logbook)
+    get_item("#{logbook} logbook")
+    case bput("bundle my #{@noun} with my logbook", 'You notate the', 'This work order has expired', 'The work order requires items of a higher quality')
```

Comment:
Does `@noun` work here? I thought this would need to be a parameter to the function...